site/design/index.html

```diff
@@ -309,7 +309,7 @@ <h2>Abteilung</h2>
 
         <ul class="value-grid">
             <li class="item">
-                <div class="key">
+                <div>
```

Comment:
why?
Just use the same html als everywhere else. removing this class breaks it on mobile.

Reply:
added the class key